moninag/static/src/ts/app/app-routing.module.ts

```diff
@@ -73,6 +74,7 @@ const APP_ROUTES: Routes = [
     imports: [
         HttpModule,
         JsonpModule,
+        FlexLayoutModule.forRoot(),
```

Comment:
Delete `FlexLayoutModule.forRoot()` and `HttpModule, JsonpModule`.
They are imported already in app.module.ts too :repeat_one: